wpengine/faustjs
# MERL-1339: add WP notice for anonymous telemetry opt-in

## Tasks


## Description

[MERL-1339](https://wpengine.atlassian.net/browse/MERL-1339)

## Testing
- Check out this branch, navigate to the Faust settings page under Settings > Faust.
- You should see a notice asking you to opt in.
- Use the buttons to test the functionality for toggling the setting.
- You should also see a checkbox setting under Features which controls the telemetry setting.
- Play with these various settings and ensure they work as expected. Try to break it.


[MERL-1339]: https://wpengine.atlassian.net/browse/MERL-1339?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
